src/main/kotlin/com/approvers/devlazaApi/model/Projects.kt

```diff
@@ -8,8 +8,11 @@ import javax.persistence.Column
 import javax.persistence.Entity
 import javax.persistence.GeneratedValue
 import javax.persistence.Id
+import javax.persistence.Table
+import javax.persistence.Index
 
 @Entity
+@Table(indexes = [Index(name="projects_index", columnList="NAME, ID, RECRUITING", unique=true)])
```

Comment:
```suggestion
@Table(indexes = [
    Index(columnList="name"),
    Index(columnList="recruiting")
])
```
元のやつだとNAME, ID, RECRUITINGのペアのインデックスだからこれ全部揃ってないと仕事しない(気がする)

Reply:
あとidのインデックスは自動で貼られるのでいらない

Reply:
了解した